src/test/kotlin/org/partiql/scribe/targets/SqlTargetSuite.kt

```diff
@@ -113,6 +106,13 @@ abstract class SqlTargetSuite {
                             }
                         }
                     }
+                    comparator.assertEquals(expected, actual) {
```

Comment:
Moving the equality check after the error checking made it a lost easier to debug Scribe errors. Otherwise, the unequal query strings would throw first and we would not be able to see the errors.